lang/en/tool_policy.php

```diff
@@ -40,6 +40,7 @@
 $string['agreed'] = 'Agreed';
 $string['agreedby'] = 'Agreed by';
 $string['agreedon'] = 'Agreed on';
+$string['agreetoselected'] = 'Agreed to selected';
```

Comment:
Should it be "Agree to selected" or even better "Agree on behalf of selected users" or so (which could be also re-used as the modal submit button text)

Reply:
> should also be "Agree"

Sorry, yes, that was the point.